orc8r/cloud/go/tools/combine_swagger/spec/combine.go

```diff
@@ -17,187 +17,54 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"regexp"
-	"sort"
 	"strings"
-
-	"magma/orc8r/cloud/go/tools/swaggergen/generate"
-
-	"github.com/hashicorp/go-multierror"
-	"github.com/pkg/errors"
-	"gopkg.in/yaml.v2"
 )
 
-// Load specs from file to Swagger structs.
-func Load(inDir, commonFilepath string) ([]generate.SwaggerSpec, generate.SwaggerSpec, error) {
+// Load specs from file to YAML string
```

Comment:
- End doc sentences with punctuation
- Document the multi-return -- specs + common